lib/pages/settings_page.dart

```diff
@@ -78,7 +83,32 @@ class _SettingsPageState extends State<SettingsPage> {
               textAlign: TextAlign.center,
               style: Theme.of(context).textTheme.headline1),
           SizedBox(height: 10),
-          Flexible(child: settingsWidget)
+          Container(height: 330, width: 500, child: settingsWidget),
```

Comment:
Prob not ideal to fix a height, so we'd prob have to change this height every time we add a setting thing.

Reply:
If we wrap it in flexible or expanded, it stretches to the whole screen and we'd need to add a sized box at the bottom to make the text visible. Should I do that instead?

Reply:
No. There seems to be multiple ways to do it properly. 
Make them both expanded and modify the flex property appropriately. 
Or use spaceEvenly in the column's mainAxisAlignment property.